### ISSUE TITLE
UI/UX of Admin Dashboard

### ISSUE THREAD

**Issue by philbarker** [t3-innovation-network/desm#601]
The Admin Dashboard UI looks like:

![image](https://github.com/user-attachments/assets/1a007092-9b09-4609-ba76-ba8493d032bc)

First, I like the side bar navigation

In the the top menu bar, "View Mappings" and "New Mapping" are of no use, they are not active for admin users. This should be indicated by removing or greying them (depending on what is done on rest of site for inactive buttons)

I am not clear what the radio button options do here.   The ability to filter on these would be more useful on the other screens, but they are not available for those. Either delete them or make them active on all screens.  If they are kept, please change the Active Mappings box to show the number of mappings in whatever category is selected.

Of the blue squares, the one for Agents works as a link to the Agent screen. Please make them all work like that.

**Comment by edgarf** [comment]
@philbarker I"m thinking, maybe we should have a session to review this page, and then also the inside pages as well? So that the whole section would have the same design 'feeling'?
